index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css">
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="mobile">
+      <header>
+        <a href="">My Log</a>
+        <nav>
+          <img src="./images/Vector.png" alt="image was moved to another location">
+        </nav>
+      </header>
+      <section>
+        <div class="bg">
+          <img id="Ellipse11" src="./images/Ellipse11.png" alt="ellipse11">
+          <img id="Ellipse3" src="images/Ellipse3.png" alt="ellipse3">
+          <img id="Rectangle4" src="./images/Rectangle4.png" alt="rectangle4">
+        </div>
+        <h1>I`m Joffrey NKESHIMANA, Glad to see you!</h1>
+        <p>
+          I`m a software developer! I can help you building a product, feature or
+          website look through some of my work and experience! If you like what
+          you see and have a project you need coded, don`t hesitate to contact me.
+        </p>
+        <div class="connect">
+          <p id="connect">LET`S CONNECT</p>
+          <div class="imgdown">
+            <img id="Ellipse12" src="./images/Ellipse12.png" alt="Ellipse12">
+          <img id="Ellipse13" src="./images/Ellipse13.png" alt="Ellipse13">
+          </div>
```

Comment:
![image](https://user-images.githubusercontent.com/38088818/164760613-0e7bf23e-c43e-4365-9eff-d7932329a666.png)
[OPTIONAL] Kindly consider reducing the spacing between your `description` text, `let's connect` text, and `social icons` so that it is as close to [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as possible:

![image](https://user-images.githubusercontent.com/38088818/164761154-334ae545-db7f-4c4a-b819-4a981a5c46d4.png)